src/analyzer/protocol/tcp/TCP_Reassembler.h

```diff
@@ -105,10 +105,10 @@ class TCP_Reassembler final : public Reassembler {
     bool did_EOF;
     bool skip_deliveries;
```

Comment:
Here, but also for Analyzer: Would bitfields for all those bools in these classes help to squeeze out a bit more? But then, even for 1mio concurrent TCP connections this would maybe save 4MB, maybe some more for connections with very many analyzers, but seems 10's of MB rather than 100's of them.

Reply:
It'd likely lose some performance with accessing the bitfields vs the discrete bools. Like you said the memory savings probably aren't worth it at that point.